entrypoint/versionbump/cilium_test.go

```diff
@@ -0,0 +1,94 @@
+//go:build cilium
+
+package versionbump
+
+import (
+	"fmt"
+
+	"github.com/rancher/distros-test-framework/pkg/assert"
+	"github.com/rancher/distros-test-framework/pkg/customflag"
+	"github.com/rancher/distros-test-framework/pkg/template"
+	"github.com/rancher/distros-test-framework/pkg/testcase"
+    "github.com/rancher/distros-test-framework/shared"
+
+	. "github.com/onsi/ginkgo/v2"
+)
+
+var _ = Describe("VersionTemplate Upgrade:", func() {
+	It("Start Up with no issues", func() {
+		testcase.TestBuildCluster(GinkgoT())
+	})
+
+	It("Validate Node", func() {
+		testcase.TestNodeStatus(
+			assert.NodeAssertReadyStatus(),
+			nil)
+	})
+
+	It("Validate Pod", func() {
+		testcase.TestPodStatus(
+			assert.PodAssertRestart(),
+			assert.PodAssertReady(),
+			assert.PodAssertStatus())
+	})
+
+	It("Verifies bump version on rke2 for cilium version", func() {
+		template.VersionTemplate(template.VersionTestTemplate{
+			TestCombination: &template.RunCmd{
+				Run: []template.TestMap{
+					{
+						Cmd: "sudo /var/lib/rancher/rke2/bin/crictl --config /var/lib/rancher/rke2/agent/etc/crictl.yaml images | grep cilium ," +
+                            " rke2 -v",
+						ExpectedValue:        template.TestMapTemplate.ExpectedValue,
+						ExpectedValueUpgrade: template.TestMapTemplate.ExpectedValueUpgrade,
+					},
+				},
+			},
+			InstallUpgrade: customflag.ServiceFlag.InstallUpgrade,
+			TestConfig: &template.TestConfig{
+				TestFunc:       template.ConvertToTestCase(customflag.ServiceFlag.TestConfig.TestFuncs),
+				DeployWorkload: customflag.ServiceFlag.TestConfig.DeployWorkload,
+				WorkloadName:   customflag.ServiceFlag.TestConfig.WorkloadName,
+			},
+			Description: customflag.ServiceFlag.TestConfig.Description,
+		})
+	})
+
+    It("Verifies ClusterIP Service", func() {
```

Comment:
do we want always run these tests bellow ?

Reply:
Yes, we usually do network checks after validate new cni versions